test/testimportproject.cpp

```diff
@@ -141,6 +141,25 @@ class TestImportProject : public TestFixture {
         ASSERT_EQUALS(0, importer.fileSettings.size());
     }
 
+    void importCompileCommands5() const {
+        const char json[] =
+            "[{"
+            "\"directory\": \"C:/Users/dan/git/build-test-cppcheck-Desktop_Qt_5_15_0_MSVC2019_64bit-Debug\","
```

Comment:
hmm.. we should consider writing these json testcases with raw strings instead. The escaped escapes makes it hard to read. well.. you followed the convention so I will merge this if you keep the code as it is. We could look at that afterwards.

Reply:
I'm  bit confused, how would you write it as a "raw string"?
The generated file file really has `\\` and to have it  in code both need to be escaped which makes it `\\\\`

But I'm happy to change it, I just do not understand what you mean :-)

Reply:
you should be able to copy paste the raw json, for instance:

    R"([{
              "directory": "C:/Users/dan/git/build-test-cppcheck-Desktop_Qt_5_15_0_MSVC2019_64bit-Debug",
              "command": "C:\\PROGRA~2\\MICROS~1\\201....."
              .....
         }])";

Reply:
it would have felt better with raw strings